build.gradle

```diff
@@ -1,11 +1,23 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.3.41"
+}
 apply plugin: 'java'
 
 repositories { mavenCentral() }
 
+sourceSets {
+    test {
+        resources {
+            srcDir "src/main/protelis"
+        }
+    }
+}
+
 dependencies {
-	compile("it.unibo.alchemist:alchemist:$alchemistVersion") {
-		exclude module: 'org.eclipse.xtext.dependencies'
-	}
+   compile("it.unibo.alchemist:alchemist:$alchemistVersion") {
+       exclude module: 'org.eclipse.xtext.dependencies'
```

Comment:
There should be no need to exclude this one, can you test without the exclusion?

Reply:
done